semperfiwebdesign/all-in-one-seo-pack
# Fix shortcode conflict with NextGen Albums

Issue #2774

## Proposed changes

Fix shortcode conflict with NextGen Albums to prevent album/gallery not rendering.

## Types of changes

What types of changes does your code introduce?
_Delete those that don't apply_

- Bugfix (non-breaking change which fixes an issue)
- Improves existing code

## Checklist


## Testing instructions
1. Have both AIOSEOP and NextGen (Pro) installed and activated.
2. Go to AIOSEOP General Settings and enable Schema Markup.
3. With NextGen prepare a gallery or album ([Tutorial](https://www.imagely.com/docs/albums/))
4. With the album shortcode ready, add it to a post/page.
5. After publishing, view post/page.

Expected: NextGen's album/gallery should render.  
Resolves: NextGen galleries not loading on page
